scripts/s3-sanity/s3-sanity-test.sh

```diff
@@ -210,6 +210,23 @@ fi
 
 echo "*** S3 Sanity tests start ***"
 
+echo "Replication test starts"
+cmd_out=$(ldapsearch -b "o=s3-background-delete-svc,ou=accounts,dc=s3,dc=seagate,dc=com" -x -w $ldappasswd -D "cn=sgiamadmin,dc=seagate,dc=com" -H ldap://"$end_point")
+accounts_before=$(echo $cmd_out | grep accountId | wc -l)
+END=5
+for ((a=1;a<=$END;a++))
```

Comment:
@bkirunge7 @abhilekh268 
Is it OK to be serialized or we need it concurrent (parallel)?